src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,8 +1,26 @@
-export const TodoFilter = () => (
+import { Visible } from '../../types/Visible';
+
+type Props = {
+  visible: Visible;
+  setVisible: (arg: Visible) => void;
+  filterBy: string;
+  setFilterBy: (arg: string) => void;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  visible = 'all',
```

Comment:
Assigning default values directly in the destructuring assignment is not recommended for TypeScript props. Consider setting default values outside of the destructuring or ensuring that the parent component provides these values.